metrics/lib/k8s/common.bash

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+THIS_FILE=$(readlink -f ${BASH_SOURCE[0]})
+LIB_DIR=${THIS_FILE%/*}
+RESULT_DIR="${LIB_DIR}/../results"
+
+source ${LIB_DIR}/../../lib/common.bash
+source ${LIB_DIR}/json.bash
+source /etc/os-release || source /usr/lib/os-release
+
+# Set variables to reasonable defaults if unset or empty
+RUNTIME="${RUNTIME:-containerd-shim-kata-v2}"
+cri_runtime="${CRI_RUNTIME:-containerd}"
+
+# If we fail for any reason, exit through here and we should log that to the correct
+# place and return the correct code to halt the run
+die(){
+	msg="$*"
+	echo "ERROR: $msg" >&2
+	exit 1
+}
+
+# Sometimes we just want to warn about something - let's have a standard
+# method for that, so maybe we can make a standard form that can be searched
+# for in the logs/tooling
+warning(){
+	msg="$*"
+	echo "WARNING: $msg" >&2
+}
+
+# This function checks existence of commands.
+# They can be received standalone or as an array, e.g.
+#
+# cmds=(“cmd1” “cmd2”)
```

Comment:
Aside: I don't like the way GitHub is rendering these double quotes as utf8 ones (but oddly not in the comment below!?! ;)